### PR TITLE
More improvements to equity value, offers, negotiations

### PR DESCRIPTION
**Pull request by jlevy**
Some other definitions cleanups.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,22 +176,24 @@ And what is this ownership worth?
   [several kinds of business entities](https://en.wikipedia.org/wiki/Types_of_business_entity#United_States)
   may be formed, or [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)),
   under the laws of a specific state.
-
+- 🄳 A **startup** is an emerging company that aspires to grow quickly, especially by
+  providing something-like a product or technology-that is new to a market and has the
+  potential to transform that market.
+  Once a company is established in the market and successful for a while, it usually stops
```
Comment (RacheltheEditor) [inline README.md]: These time references are vague. What does "emerging" actually mean, wrt time or size? What measurements (time? size? money? 'presence'? office space? name recognition?) define "successful" and "for a while" and "established"? Offer some constraints/specifics.

File: README.md [hunk 2]
```diff
@@ -360,37 +354,50 @@ And what is this ownership worth?
 
 ### How does the value of equity change?
 
-We’ve now seen how complex it is to assess the value of equity in private companies.
-Well, it doesn’t get any easier, because if a company is giving you equity, the intention
-is to grow the company, and so the value of your equity is always changing over time.
+We’ve now seen how hard it is to evaluate the value of equity in private companies,
```
Comment (RacheltheEditor) [inline README.md]: "determine" the value

File: README.md [hunk 3]
```diff
@@ -498,13 +493,15 @@ Some details on how this works:
...
 - 🄳
   [**Liquidation overhang**](https://equityzen.com/blog/startup-valuations-and-liquidation-preference-overhang/)
-  can occur when the value of a company just doesn’t reach the number of dollars investors
-  put into it. Because of liquidation preference, those holding preferred stock (investors)
-  will have to be paid before those holding common stock (employees).
-  If investors invested a hundred million dollars in your company, your equity as an
-  employee won’t be worth anything if the company is sold when it’s in liquidation overhang
-  and the sale doesn’t
-  [exceed that amount](https://avc.com/2010/10/employee-equity-the-liquidation-overhang/).
+  is a situation where the value of a company just doesn’t reach the number of dollars
```
Comment (RacheltheEditor) [inline README.md]: "is a situation" and "for example" are almost never needed phrases. Would revert these changes!

File: README.md [hunk 4]
```diff
@@ -1189,10 +1186,31 @@ negotiation.
...
+You’re planning to devote a lot of your time and life to any full-time role, and it’s
+worth making sure it’s a good investment of that time.
+
+☝️ A few people are inexperienced, uncertain, or so anxious about negotiations they avoid
+them as much as they can, sometimes just accepting an offer with little discussion.
+It can indeed be stressful to discuss offers.
+However, even if you feel uncomfortable negotiating an offer with a potential employer, it
+may help to remember why it’s important to have these conversations:
+
+- Firstly, negotiations are important to let you focus on what you actually want.
```
Comment (RacheltheEditor) [inline README.md]: Ditto "first," "second," "lastly."

File: README.md [hunk 5]
```diff
@@ -1189,10 +1186,31 @@ negotiation.
...
+  Not being clear with yourself on what things are important to you, be it personal growth,
```
Comment (RacheltheEditor) [inline README.md]: Ditto "be it"

File: README.md (hunk 2 above)
Comment (RacheltheEditor) [inline README.md]: Surprisingly common stylistic issue so I'll mention it here (though Josh I can take care of these changes after merge if need, just noting stuff): verbs should never contain the object they act upon, e.g. "evaluate the value." Try "determine" or "establish"

File: README.md [hunk 6]
```diff
@@ -1189,10 +1186,31 @@ negotiation.
...
+  career growth, impact, recognition, cash, ownership, or teamwork.
+- Rushing through this can actually shortchange not just yourself, but others you will work
```
Comment (RacheltheEditor) [inline README.md]: Adjust the passive--unnecessarily complex phrasing ("Rushing through this..."). Just tightening these three sentences solves. "If you aren't satisfied with the terms of your offer, accepting it can be tough not just for you but for your company and colleagues as well. No one wants to take on a hire who's going to walk away when something better comes along. For everyone's sake, take your time now to consider what you really want--and then ask for it."

File: README.md [hunk 7]
```diff
@@ -1217,6 +1235,73 @@ candidates. But even so, sophisticated negotiators can often get better offers.
...
+company, and the role, so you make a good decision.
+
+Things to look for in the offer letter include:
+
+- Title and level:
+  What your role is officially called, who you report to, and what level of seniority your
+  role is within the company.
+- Salary: What you’re paid.
+- Equity compensation:
+  You know what this is now.
```
Comment (RacheltheEditor) [inline README.md]: lol

File: README.md [hunk 8]
```diff
@@ -1217,6 +1235,73 @@ candidates. But even so, sophisticated negotiators can often get better offers.
...
+manager, your future manager, or a recruiter, or some combination) multiple times before
+signing. This helps you negotiate details and also just get to know the people, the
+company, and the role, so you make a good decision.
+
+Things to look for in the offer letter include:
+
+- Title and level:
+  What your role is officially called, who you report to, and what level of seniority your
+  role is within the company.
+- Salary: What you’re paid.
```
Comment (RacheltheEditor) [inline README.md]: "in a year, before taxes"

File: README.md [hunk 9]
```diff
@@ -1217,6 +1235,73 @@ candidates. But even so, sophisticated negotiators can often get better offers.
...
+- Accommodations:
+  Other aspects of the job that might matter to you, like time off, ability to work from
+  home, flexible hours, and so on.
+
+Some general notes on these components, including
+[several by Cristina Cordova](https://twitter.com/cjc/status/984094472190349312):
+
+- Early stage startups will focus on salary and equity and (if they are funded) benefits.
+  Bonus and signing bonus are more common in larger cash-flush companies.
+- Bonus is usually standardized to the company and your level, so not likely to be something
```
Comment (RacheltheEditor) [inline README.md]: Is it a thing to refer to "bonus" rather than "bonuses"?

File: README.md [hunk 10]
```diff
@@ -1217,6 +1235,73 @@ candidates. But even so, sophisticated negotiators can often get better offers.
...
+  Depending on terms you may need to pay it back if you leave the company right away.
+
+### Startup offers
+
+Because startups are so much smaller than many established companies, and because they may
+grow quickly, a few other things are worth remembering when negotiating with startups:
+
+- **Cash versus equity**: If your risk tolerance is reasonably high, it’s reasonable to ask for
+  an offer with more equity and less cash.
+  Usually if a company is doing well, it is likely to “level up” lower salary levels later,
```
Comment (RacheltheEditor) [inline README.md]: meaning, if you accept a salary below market now, you can usually expect to have your salary raised as the company grows

File: README.md [hunk 11]
```diff
@@ -1217,6 +1235,73 @@ candidates. But even so, sophisticated negotiators can often get better offers.
...
+  even if you got more equity up front.
+  On the other hand, if you ask for more cash and less equity, it’s unlikely you’ll get all
+  that equity back.
+- Negotiating title and exact details of your role front may not matter as much in a small
```
Comment (RacheltheEditor) [inline README.md]: Why

**Comment by RacheltheEditor** [comment]
Don't worry about grammar I've commented on. There are typos here as well I can correct after merge.  @jlevy just address questions plz!

File: README.md (hunk 1 above)
Comment (RacheltheEditor) [reply README.md]: Also, what is "trade"

Comment (AndySparks) [reply README.md]: I see that you've used the language from *Raising Venture Capital*. One dominant argument ([from Paul Graham](http://www.paulgraham.com/growth.html)) is that a startup is "a company designed to grow fast." Maybe start with what a startup *is not*. A startup is not:
* A "lifestyle business" (small "mom and pop" shop with no ambitions to grow beyond 1-2 locations)
* A public company
* A private company with $1-100M+ in revenue that is profitable but not growing YoY

I'm with Rachel that we should recognize that it's a word without a broadly agreed upon definition. Additionally, I think we should emphasize that startups are different from a lot of other businesses in that they not only need to attract *really stellar* talent in order to tackle a big problem quickly, *they need to incentivize people to come work for them when the business is cash poor.*

File: README.md (hunk 3 above)
Comment (jlevy) [reply README.md]: Sure re "for example" (though in general we probably should tag detailed examples).

Regarding "is a situation" the main thing is this definition was not a definition before, but a more vague statement saying it's something that "occurs". A few definitions are like this and we should clean that up.

Comment (RacheltheEditor) [reply README.md]: Rather, the previous definition indicated that it was 'something that occurs when'... i.e., a "situation," 'something that occurs'

File: README.md (hunk 1 above)
Comment (RacheltheEditor) [reply README.md]: The "what it's not" tact is very helpful! Nice work, @AndySparks

File: README.md (hunk 6 above)
Comment (jlevy) [reply README.md]: 👍 Fixed.

File: README.md (hunk 8 above)
Comment (jlevy) [reply README.md]: Fixed.

File: README.md (hunk 11 above)
Comment (jlevy) [reply README.md]: Fixed.

File: README.md [hunk 12]
```diff
@@ -1217,6 +1235,73 @@ candidates. But even so, sophisticated negotiators can often get better offers.
...
+### Startup offers
+
+Because startups are so much smaller than many established companies, and because they may
+grow quickly, a few other things are worth remembering when negotiating with startups:
+
+- **Cash versus equity**: If your risk tolerance is reasonably high, it’s reasonable to ask for
+  an offer with more equity and less cash.
+  Usually if a company is doing well, it is likely to “level up” lower salary levels later,
+  even if you got more equity up front.
+  On the other hand, if you ask for more cash and less equity, it’s unlikely you’ll get all
```
[the remark this answers is not in the document]
Comment (jlevy) [reply README.md]: Fixed.

File: README.md [hunk 13]
```diff
@@ -1292,20 +1379,34 @@ it, at least not by much.
...
+
+- 🔹 Although offer letters have expirations, it’s often possible to negotiate more time if
+  you need it. How much flexibility depends on the situation.
+  Some people will go so far as to say “exploding job offers”
+  [are a bad practice](https://www.huffingtonpost.com/adam-grant/its-time-to-eliminate-exp_b_4594222.html)
+  or
+  [make no sense at all](https://erikbern.com/2016/03/16/exploding-offers-are-bullshit.html). If
+  you are likely the best candidate for the position, or role is a specialized and well-paid
+  one where there are usually not enough good candidates to meet the demand, you’ll likely
+  have plenty of “leverage” to ask for more time, such as to complete the interview process
```
[the remark this answers is not in the document]
Comment (jlevy) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (jlevy) [reply README.md]: Fixed.

File: README.md (hunk 5 above)
Comment (jlevy) [reply README.md]: REwritten.

File: README.md [hunk 14]
```diff
@@ -576,37 +584,35 @@ well for you—and when it might not.
...
+  - For example, if your equity award had a one-year cliff and you only worked for the company
+    for 11 months, you would not get anything, since you haven’t vested in any part of your
+    award. Similarly, if the company is sold within a year, depending on what your paperwork
+    says, you may also receive nothing on the sale of the company.
+  - A very common vesting schedule is vesting over **4 years**, with a **1 year** cliff.
+    This means you get 0% vesting for the first 12 months, 25% vesting at the 12th month, and
+    1/48th (2.08%) more vesting each month until the 48th month.
+    If you leave just before a year is up, you get nothing, but if you leave after 3 years,
+    you get 75%.
+  - 🌪 Cliffs are an important topic.
```
Comment (jlevy) [inline README.md]: I think I'll move this back to the top. I don't think we can cover everything about cliffs as a sub-bullet of the definition. It's too big. Arguably we should have a whole subsection on cliffs.

Comment (RacheltheEditor) [reply README.md]: Maybe we do. It's not easy to be introduced to a concept and then learn more about it later on when you thought you'd already been told what's important. So if you move it down but we don't create a new subsection, just remember that transitional sentences can be lifesavers--and are needed a lot in this Guide, where information is spread out. Simple additions like, "We mentioned cliffs, and there are a few more details you should know..." These really help the reader not feel like they just missed some connection.

File: README.md [hunk 15]
```diff
@@ -852,7 +904,8 @@ calculated.
     that if you move to a higher bracket, you’ll make less money.
     What actually happens is when you cross certain thresholds, each additional (marginal)
     dollar money you make is taxed at a higher rate, equal to the bracket you’re in.
-    It looks roughly [like this](http://i.imgur.com/uLaJJ5q.png) ([source](https://www.reddit.com/r/personalfinance/comments/2wkbgz/graphing_one_misconception_about_tax_brackets/)).
+    It looks roughly [like this](http://i.imgur.com/uLaJJ5q.png), created from a ([Reddit thread](https://www.reddit.com/r/personalfinance/comments/2wkbgz/graphing_one_misconception_about_tax_brackets/)
```
Comment (jlevy) [inline README.md]: Fyi, looks like parens misplaced.

File: README.md [hunk 16]
```diff
@@ -272,79 +255,87 @@ And what is this ownership worth?
...
   - 🔸 Sales generally require the agreement and cooperation of the company, for both
     contractual and practical reasons.
-    - 🄳 Typically shares held are subject to a **right of first refusal** in favor of the
-      company, meaning someone can’t sell their shares to a third party without offering to sell
-      it to the company first.
-  - **Why companies may not support secondary sales:**
-    It’s important to consider a company’s perspective on secondary sales as well:
+    - 🄳 Typically, shares held by an employee are subject to a **right of first refusal** in
+      favor of the company, meaning someone can’t sell their shares to a third party without
+      offering to sell to the company first.
```
Comment (AndySparks) [inline README.md]: Awkward phrasing WRT what is being sold. Too easy to mistake "offering to sell" with a company being sold. I propose an alternate for clarity:
"Typically, shares held by an employee are subject to a **right of first refusal** in favor of the company, meaning the employee can't sell their shares to a third party without offering to sell their shares to the company first."

File: README.md [hunk 17]
```diff
@@ -536,23 +553,28 @@ In this section we’ll lay out the differences, benefits, and drawbacks of comm
...
+- 🔸 Options are not the same as stock;
+  they are only the right to buy stock upon and subject to the conditions specified in the
+  option agreement.
 
-🚧 Add a tip about how this isn’t at all the same as trading in options on stocks.
+🚧 Add a tip about how this isn’t at all the same as trading in options on stocks, or
+buying stock options (differentiate ESOs as a type of stock option)
 
-Stock options allow you to buy shares at a *fixed* price per share, the **strike price**.
+🄳 The **strike price** is the *fixed* price per share that stock options allow.
```
Comment (AndySparks) [inline README.md]: Awkward phrasing. Alternate:

The **strike price** is the *fixed* price per share at which stock options can be exercised, as set in an employee's stock option agreement.

**Comment by jlevy** [comment]
@RacheltheEditor I've fixed all the formatting — do review so you can see the issues. And the lost changes from earlier, which I'm not sure if you reviewed.

But given this is now in workable shape I'm merging and starting work on a fresh PR.
